### PR TITLE
Added LinSpace to Raw_Ops for tf frontends

### LINKED ISSUE #10842: LinSpace

### PR DESCRIPTION
**Pull request by sdk2k01**
Closes #10842

### CODE REVIEW

File: ivy/functional/frontends/tensorflow/raw_ops.py [hunk 1]
```diff
@@ -607,3 +607,9 @@ def Elu(features, name=None):
         "float64",
     ),
 }
+
+
+# @with_unsupported_dtypes({"2.9.1 and below": ()}, "tensorflow")
```
Comment (sherry30) [inline ivy/functional/frontends/tensorflow/raw_ops.py:612]: commented code

File: ivy/functional/frontends/tensorflow/raw_ops.py [hunk 2]
```diff
@@ -607,3 +607,9 @@ def Elu(features, name=None):
...
+@to_ivy_arrays_and_back
+def LinSpace(*, start, stop, num, name=None):
```
Comment (sherry30) [inline ivy/functional/frontends/tensorflow/raw_ops.py:614]: These arguments are not keyword-only, you can remove `*`

**Review by sherry30 — CHANGES_REQUESTED** [review]
Hi,

Your PR looks good, just few changes I've requested on specific lines.
I'll merge your PR as soon as you make these changes.

Thanks 😄

File: ivy/functional/frontends/tensorflow/raw_ops.py (hunk 1 above)
Comment (sdk2k01) [reply ivy/functional/frontends/tensorflow/raw_ops.py:612]: Hi! thanks for your quick response. I will correct this.

File: ivy/functional/frontends/tensorflow/raw_ops.py (hunk 2 above)
Comment (sdk2k01) [reply ivy/functional/frontends/tensorflow/raw_ops.py:614]: As for this, the test is now failing with an error as follows: 
```sh
E       TypeError: lin_space only takes keyword args (possible keys: ['start', 'stop', 'num', 'name']). Please pass these args as kwargs instead.
../../mambaforge/envs/ivy_dev/lib/python3.8/site-packages/tensorflow/python/util/tf_export.py:396: TypeError
```

File: ivy/functional/frontends/tensorflow/raw_ops.py (hunk 1 above)
Comment (sdk2k01) [reply ivy/functional/frontends/tensorflow/raw_ops.py:612]: Hi! thanks for your quick response on the PR! I have made the changes as requested. However, for [another requested change](https://github.com/unifyai/ivy/pull/10871#discussion_r1115842539), I'm encountering an issue you might want to take a look at.

File: ivy/functional/frontends/tensorflow/raw_ops.py (hunk 2 above)
Comment (sdk2k01) [reply ivy/functional/frontends/tensorflow/raw_ops.py:614]: Here's a [related error](https://stackoverflow.com/questions/65257591/tensorflow2-placeholder-only-takes-keyword-args) I found on the issue, if that helps (apologies if I'm being wrong somewhere).

Comment (sherry30) [reply ivy/functional/frontends/tensorflow/raw_ops.py:614]: Oh, You're right, I wasn't aware of that. 
Thanks for clearing that up 😄

**Review by sherry30 — APPROVED** [review]
Looks good!

Thanks for the contribution. 👍

**Pull request merged**